doc-site/docs/tutorials/notarized-tokens.md

```diff
@@ -0,0 +1,135 @@
+# Notarized Tokens
+
+In this tutorial, you’ll learn how to create and manage a basic token using the **Notarized Tokens (noto)** domain. Unlike the private storage example, these tokens can be transferred between nodes publicly, demonstrating how assets (e.g., “cash”) can be issued and tracked on the blockchain using Paladin.
+
+## Prerequisites
+
+Make sure you have:
+
+1. Completed the [Private Storage Tutorial](./private-storage.md).
+2. A **running Paladin network** with at least three nodes (Node1, Node2, Node3).
+
+---
+
+## Overview
+
+This tutorial will guide you through:
+
+1. **Deploying a Noto Token**: Use the `NotoFactory` to create a “cash token” with a specific notary.
+2. **Minting Tokens**: Issue tokens to a particular node’s account.
+3. **Transferring Tokens**: Send tokens between different nodes to simulate basic payments.
+
+You can find the complete example code in the [Paladin example repository](https://github.com/LF-Decentralized-Trust-labs/paladin/tree/main/example/notarized-tokens).
+
+---
+
+## Step 1: Deploy a Noto Token
+
+First, create a **Noto Factory** instance and deploy a new token. In this scenario, Node1 will act as both the notary (the entity allowed to mint tokens) and the initial recipient of the minted cash.
+
+```typescript
+logger.log("Step 1: Deploying a Noto cash token...");
+const notoFactory = new NotoFactory(paladinClientNode1, "noto");
+const cashToken = await notoFactory.newNoto(verifierNode1, {
+  notary: verifierNode1,          // The notary for this token
+  restrictMinting: true,          // Restrict minting to the notary only
+});
+if (!cashToken) {
+  logger.error("Failed to deploy the Noto cash token!");
+  return false;
+}
+logger.log("Noto cash token deployed successfully!");
+```
+
+**Key Points**:
+- **`notary`**: Specifies which verifier (account) can mint new tokens.
+- **`restrictMinting`**: If `true`, only the `notary` can mint additional tokens.
+
+---
+
+## Step 2: Mint Tokens
+
+Now that the token contract exists, **mint** an initial supply of tokens to Node1. This step simulates creating new “cash” in the system.
+
+```typescript
+logger.log("Step 2: Minting 2000 units of cash to Node1...");
+const mintReceipt = await cashToken.mint(verifierNode1, {
+  to: verifierNode1,              // Mint cash to Node1
+  amount: 2000,                   // Amount to mint
+  data: "0x",                     // Additional data (optional)
+});
+if (!mintReceipt) {
+  logger.error("Failed to mint cash tokens!");
+  return false;
+}
+logger.log("Successfully minted 2000 units of cash to Node1!");
+```
+
+**Key Points**:
+- **`amount`**: Number of tokens to create.
+- **`data`**: Can include extra metadata or encoding, if needed.
```

Comment:
The data will be recorded in the domain receipt.
(this is helpful to understand why you would want to add data)